src/test/java/org/ays/payload/Helper.java

```diff
@@ -502,6 +500,16 @@ public static AdminCredentials setIntsAdminCredentials() {
         return adminCredentials;
     }
 
+    public static List<Map<String, String>> extractResponseAsList(Response response) {
+        JsonPath jsonPath = response.jsonPath();
+        return jsonPath.getList("response");
+    }
+
+    public static void customLog(String message) {
+        log.info(message);
+        Reporter.log(message);
+    }
```

Comment:
Burayı böyle yapmamamız daha iyi olabilir, farklı bir yapı kuralım. Bir de Helper sınıfı her işi yapınca çok karışıyor. Biraz daha spesifik sınıflar oluşturmamız laızm.

Reply:
Helper class icin ayrı bir issue acmistik, onun uzerinden Helper düzenlemesi yapsak olur mu? Şimdilik bu boyle kalsın ya da test class içerisine mi alayım. Ne tasviye edersin?

Reply:
Helper Class için açtığımız issue'yu yapana kadar her şeyi Helper Class'a eklersek düzenlemesi çok zaman alır. O yüzden yeni eklediklerimizi buraya eklemek yerine mesela AysLogUtil gibi bir sınıf açıp oraya ekleyelim bu tarz şeyleri.